straxen/get_corrections.py

```diff
@@ -114,72 +93,44 @@ def get_to_pe(run_id, gain_model, n_pmts):
 @export
 @correction_options
 def get_correction_from_cmt(run_id, conf):
+    """
+    Get correction from CMT
+    :param run_id: run id from runDB
+    :param conf: configuration 
+    :return: correction
+    """
+ 
     if isinstance(conf, str) and conf.startswith('https://raw'):
         # Legacy support for pax files
         return conf
     if isinstance(conf, tuple) and len(conf) == 3:
         is_nt = conf[-1]
-
-        model_type, global_version = conf[:2]
+        model_conf, global_version = conf[:2]
         correction = global_version  # in case is a single value
```

Comment:
Could we just rename it in `global_version_or_constant_value` ? In the current version it is super hard to understand because it is very inconsistently used.  
In line 111 you check if global_version is a float or int while the setting "global_version" should be a string if specified as version. 
In line 117 you overwrite correction with a correction value etc.

Reply:
fixed